test/puppeteer_utils.js

```diff
@@ -374,4 +376,45 @@ describe('Apify.utils.puppeteer', () => {
             await browser.close();
         }
     });
+
+    it('saveSnapshot() works', async () => {
+        const mock = sinon.mock(keyValueStore);
+        const browser = await Apify.launchPuppeteer({ headless: true });
+        try {
+            const page = await browser.newPage();
+            const contentHTML = '<html><head></head><body><div style="border: 1px solid black">Div number: 1</div></body></html>';
+            await page.setContent(contentHTML);
+
+            const screenshot = await page.screenshot({ fullPage: true });
```

Comment:
Let's do JPEGs to save some space and CPU with 65% quality or something that will be readable but small

Reply:
So I have settled on 50%. 

PNG has 1.3MB
50% - 340 KB
10% - 150 KB - but it was too blurry I think